sdk/exportentities/action.go

```diff
@@ -309,22 +309,20 @@ func (s Step) AsScript() (*sdk.Action, bool, error) {
 	}
 
 	bS, ok := bI.(string)
-
 	if !ok {
-		var arScript []interface{}
-		arScript, ok = bI.([]interface{})
-		var asScriptString = make([]string, len(arScript))
-		for i, s := range arScript {
-			asScriptString[i], ok = s.(string)
+		asScript, ok := bI.([]interface{})
```

Comment:
there is 3 different variable named `ok` here. Can you choose a different name for each please ?

Reply:
done